admin/client/src/components/GridField/GridField.scss

```diff
@@ -58,41 +63,43 @@
 .grid-field {
 
   .ss-gridfield-sort {  // Todo: rename to grid-filed__sort
-      background: transparent url(../images/arrows.png) no-repeat right -1px;
-      border: none;
-      width: 100%;
+    background: transparent url(../images/arrows.png) no-repeat right -1px;
+    border: 0;
+    width: 100%;
+    padding: 0;
+    text-shadow: none;
+    border-radius: 0;
+    text-transform: uppercase;
+    font-weight: normal;
+    text-align: left;
+
+    .ui-button-text {
       padding: 0;
-      text-shadow: none;
-      border-radius: 0;
-      text-transform: uppercase;
-      font-weight: normal;
-      text-align: left;
-
-      .ui-button-text {
-        padding: 0;
-        line-height: 20px;
-        color: $body-color-light;
-      }
-
-      &:hover {
-        box-shadow: none;
-        background: transparent url(../images/arrows.png) no-repeat right -41px;
-      }
-      &.ss-gridfield-sorted-asc,
-      &.ss-gridfield-sorted-asc:hover {
-        background-position-y: -118px;
-      }
-      &.ss-gridfield-sorted-desc,
-      &.ss-gridfield-sorted-desc:hover {
-        background-position-y: -78px;
-      }
+      line-height: 20px;
+      color: $body-color-light;
+    }
+
+    &:hover {
+      box-shadow: none;
+      background: transparent url(../images/arrows.png) no-repeat right -41px;
+    }
+
+    &.ss-gridfield-sorted-asc,
+    &.ss-gridfield-sorted-asc:hover {
+      background-position-y: -118px;
+    }
+
+    &.ss-gridfield-sorted-desc,
+    &.ss-gridfield-sorted-desc:hover {
+      background-position-y: -78px;
+    }
   }
 
-  thead tr.title th {
```

Comment:
.title isn't used currently anywhere with the new gridfield but if you apply the new gridfield class to the older grid-field it will get the new styles. Kind of a work in progress.